### ISSUE TITLE
Local crop loss 

### ISSUE THREAD

**Issue by EmanuelSoda** [marrlab/DinoBloom#1]
Hi Marr Lab,

First of all, congratulations on the outstanding preprint and the impressive model you developed!

I am currently training a DINOv2 model on centered objects, which is quite similar to your work. While reading your manuscript, I noticed that you removed the local crop loss, as mentioned:

> "We remove the global-local crop loss as we found it hampers performance when learning representations on the single cell images in blood and bone marrow datasets."

I am experiencing a high local crop loss and noticing that the model pays excessive attention to small details in the objects, rather than focusing on the general shape, which would be more beneficial in my case.

Could you please share your observations during training that led you to adopt this different training strategy? Any insights or hints you can provide would be greatly appreciated.

Thank you very much for your time and assistance!

Best regards,
Emanuel

**Comment by EmanuelSoda** [comment]
Hi Valentin,

Apologies for the delayed response. Yes, that would be incredibly helpful.

I’ve actually applied the strategy you used for DinoBloom to my own case and have reached the point where I need to compare the two approaches. So far, it seems that the loss trend is consistent in both cases, but I’m observing a higher global loss value with the DinoBloom strategy.

I understand that interpreting losses in self-supervised learning isn’t always straightforward, but I was wondering if you’ve encountered a similar situation and how you went about evaluating your model. 

Looking forward to your insights.

Best regards,
Emanuel

**Comment by ValentinKoch** [comment]
Hi Emanuel,

i spent now quite some time with the DINOv2 framework, for hematology and histopathology. One thing I have observed is that loss values do not matter at all and can not predict which model will perform better in a downstream task. Doesnt matter which loss it is, whether global crop, local crop, ibot or other custom losses. Thats why my advice would be to always test your models on the downstream tasks that matter for your task. Its a bit unsatisfactory, but I would claim that noone really understands well whats going on during training to make robust predictions. I also tried other metrics like the one introduced in "RankMe: Assessing the Downstream Performance of Pretrained Self-Supervised Representations by Their Rank" but they were also useless unfortunately.

Best 
Valentin
